sru-scripts/manual/oracle-sru

```diff
@@ -17,13 +18,34 @@ assert_expected_vars PROPOSED_SCRIPT SAMPLE_CLOUDCONFIG LP_USER SSHOPTS \
 create_samplecloudconfig SAMPLE_CLOUDCONFIG
 create_setup_proposed_script
 
+function cleanup {
+    echo "Cleaning up instance ${INSTANCE_ID}"
+    oci compute instance terminate --force --instance-id="${INSTANCE_ID}"
+}
 
 echo "### BEGIN $UBUNTU_RELEASE"
 
-# TODO: Use the Oracle CLI
-echo "Please manually launch a $UBUNTU_RELEASE instance in Oracle and enter its IP address"
-echo "(Don't forget to add your SSH public key to the instance when you launch it.)"
-read -p "$UBUNTU_RELEASE IP address: " VM_IP
+declare -A releases
+releases=( ["xenial"]="16.04" ["bionic"]="18.04" ["focal"]="20.04" )
+
+IMAGE_ID=$(oci compute image list --operating-system="Canonical Ubuntu" --operating-system-version="${releases[${UBUNTU_RELEASE}]}" --sort-by="TIMECREATED" --limit=1 2>/dev/null | jq -r '.data[0]["id"]')
+VCN_ID=$(oci network vcn list | jq -r '.data[0]["id"]')
+SUBNET_INFO=$(oci network subnet list --vcn-id="${VCN_ID}" | jq '.data[0] | {id: .id, "availability-domain": ."availability-domain"}')
+SUBNET_ID=$(echo ${SUBNET_INFO} | jq -r '.id')
+AVAILABILITY_DOMAIN=$(echo ${SUBNET_INFO} | jq -r '."availability-domain"')
+
+INSTANCE_ID=$(oci compute instance launch --availability-domain="${AVAILABILITY_DOMAIN}" --shape="VM.Standard1.1" --image-id="${IMAGE_ID}" --subnet-id="${SUBNET_ID}" --ssh-authorized-keys-file="${SSH_KEY}" | jq -r '.data["id"]')
+
+trap cleanup EXIT
+
+status=$(oci compute instance get --instance-id="${INSTANCE_ID}" | jq -r '.data."lifecycle-state"')
+while [ "${status}" != "RUNNING" ]; do
+    sleep 5
+    status=$(oci compute instance get --instance-id="${INSTANCE_ID}" | jq -r '.data."lifecycle-state"')
+done
+
+oci compute instance list-vnics --instance-id="${INSTANCE_ID}"
+VM_IP=$(oci compute instance list-vnics --instance-id="${INSTANCE_ID}" | jq -r '.["data"][0]["public-ip"]')
 
 while ! ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- cloud-init status --wait --long; do
```

Comment:
this might be aided by using the helper function from sru-vars.template 
```suggestion
retry_ssh_command ubuntu@$VM_IP "loud-init status --wait --long
```